modules/scheduler-api/src/main/resources/ical4j.properties

```diff
@@ -0,0 +1,16 @@
+# Configuration file for ical4j v2.2.0
```

Comment:
@karendolan Thanks for updating ical4j to version 3.0.2. You seem to have overlooked this file...